src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,311 @@ paths:
         500:
           description: "Internal Server Error"
 
+  /alert-subscriptions:
+    post:
+      operationId: "createAndSubscribeToAnAlert"
+      tags:
+        - "Alert subscriptions"
+      summary: Create an alert and subscribe users
+      description: This request associates a query to a newly created alert using an assetId property, and subscribes users to alerts for that query through the use of emailIds.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-service-token'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
+
+        - name: alert subscription
+          in: body
+          description: Create an alert for a particular query or schedule ID and subscribe emailIds to different notification channels.
+          required: true
+          schema:
+               $ref: '#/definitions/subscribe_user_to_alert_body'
+
+      responses:
+        200:
+          description: The subscription to an alert was successful.
+          schema:
+            $ref: '#/definitions/alert_subscription_response_body'
+        400:
+          description: "Bad Request. The payload may be malformed, the alert might be in a disabled state, or an emailID provided for subscribing to the alert might not be a registered Adobe ID."
+          schema:
+            $ref: '#/definitions/post_bad_request_error_message'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "Not found. The request cannot be processed because the asset ID provided is invalid."
+          schema:
+            $ref: '#/definitions/post_not_found_error_message'
+        500:
+          description: "Internal Server Error"
+    get:
+      operationId: "listAllAlertsPerImsOrgAndSandbox"
+      tags:
+        - "Alert subscriptions"
+      summary: Retrieve all alerts present in an organization sandbox
```

Comment:
Couple points:

1. All Platform API operations take place within an organization, and nearly all of them within a sandbox as well. We don't need to mention them in the description here as this should be a given. Please comb through the swagger updates and make sure we're not unnecessarily bringing organization/sandbox into the equation unless there's something else particular we need to bring up about them.
2. Have you confirmed that this call can only return _all_ alerts for the given sandbox? If there are other query parameters one can use to filter results, "all" is misleading for a listing call. Usually, the summary for these methods are just "List Xs"